### PR TITLE
various monorepo updates

### PR DESCRIPTION
**Pull request by joanise**
We don't have CI working yet, but these changes are good, I'd like to merge them into `dev.monorepo`, so that the images and state-fix branches can be rebased onto this work.

### CODE REVIEW

**Comment by joanise** [comment]
@roedoejet This is ready to review and merge.

**Comment by joanise** [comment]
The unit tests fail because an actual test case fails, but they do actually run.

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "packages/Studio-Web"]
 	path = packages/Studio-Web
-	url = git@github.com:ReadAlongs/Studio-Web.git
+	url = ../Studio-Web.git
```
Comment (roedoejet) [inline .gitmodules:3]: good catch, thanks @joanise

File: docker-notes [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Locally:
+
+docker run --rm -it ubuntu
+
+Inside the image:
+
+apt-get update
+apt install curl git vim-nox less
+apt-get install libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
```
Comment (roedoejet) [inline docker-notes:11]: any reason not to just have a Dockerfile?

File: nx.json [hunk 3]
```diff
@@ -7,7 +7,15 @@
       }
     }
   },
-  "defaultBase": "master",
+  "defaultBase": "main",
```
Comment (roedoejet) [inline nx.json:10]: Was this actually necessary in the end?

File: packages/ngx-web-component/package.json [hunk 4]
```diff
@@ -16,12 +16,12 @@
     },
     "private": false,
     "nx": {
-        "implicitDependencies": ["@readalongs/web-component"]
+        "implicitDependencies": ["web-component"]
```
Comment (roedoejet) [inline packages/ngx-web-component/package.json:19]: ARGGGGG

File: packages/web-component/src/index.ds.ts [hunk 5]
```diff
@@ -0,0 +1,21 @@
+export interface Page {
```
Comment (roedoejet) [inline packages/web-component/src/index.ds.ts:1]: Why are these moved to this location?

**Review by roedoejet — APPROVED** [review]
Looks good to me, thanks @joanise ! there is a lot of blood, sweat, and tears behind these (relatively) few changes. Please review my comments/questions before merging.

File: docker-notes (hunk 2 above)
Comment (joanise) [reply docker-notes:11]: I would have to write and test it...

Comment (joanise) [reply docker-notes:11]: I mean, no. :)

File: nx.json (hunk 3 above)
Comment (joanise) [reply nx.json:10]: Wow, I have no recollection of this change. I don't know.

File: packages/ngx-web-component/package.json (hunk 4 above)
Comment (joanise) [reply packages/ngx-web-component/package.json:19]: Loud enough for you to hear me all the way from Ottawa!

File: packages/web-component/src/index.ds.ts (hunk 5 above)
Comment (joanise) [reply packages/web-component/src/index.ds.ts:1]: This is some refactoring Del was doing. I don't know all the details. It makes more sense with the rest of the page_transition PR.
